website/docs/r/budget.html.markdown

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
This file needs to be renamed to match the nesting: `website/docs/r/budgets_budget.html.markdown`